### PR TITLE
Use onp instead of np in ode_test

### LINKED ISSUE #3267: shouldn't this be jnp instead of np?
https://github.com/google/jax/blob/6124f703afa9c1fe6ef068741b6e6ee140b1f462/tests/ode_test.py#L45

Otherwise, `np_fun` and `jax_fun` are the same...

### PR DESCRIPTION
**Pull request by samuela**
Fixes https://github.com/google/jax/issues/3267.

### CODE REVIEW

File: tests/ode_test.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from functools import partial
 
 from absl.testing import absltest
-import numpy as np
+import numpy as onp
```
Comment (hawkinsp) [inline tests/ode_test.py:18]: We're not using this convention any more, and we are trying to be consistent about it. The convention is:
```
import numpy as np
import jax.numpy as jnp
```

Comment (mattjj) [reply tests/ode_test.py:18]: I can make this tweak!

**Comment by mattjj** [comment]
I pushed a commit to follow the new convention, and also make some other tweaks/fixes.

**Comment by mattjj** [comment]
Thanks @samuela !

**Comment by mattjj** [comment]
(Travis is green, but for some reason the GitHub UI isn't showing it.)

**Pull request merged**